### ISSUE TITLE
request: allow to show via UI designer

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [JakeWharton/ViewPagerIndicator#405]
maybe add isInEditMode , to show the indicators even in UI designer?
currently it shows nothing there...

**Comment by hackware1993** [comment]
@AndroidDeveloperLB https://github.com/hackware1993/MagicIndicator, have a try.

**Comment by AndroidDeveloperLB** [comment]
@hackware1993 Nice, but a lot of this repo isn't in English. 
Also, it doesn't get to show in the UI designer, according to what I saw:
![maxthonsnap20160728102244](https://cloud.githubusercontent.com/assets/5357526/17204435/3004e1a4-54ae-11e6-8f34-9b1b66fd9056.png)

**Comment by hackware1993** [comment]
@AndroidDeveloperLB just run demo to see results.

**Comment by AndroidDeveloperLB** [comment]
@hackware1993 This won't cause the IDE to show it. It will cause the device to show it.
The request is that I will be able to see the indicators in the IDE itself. Not the device.

**Comment by hackware1993** [comment]
@AndroidDeveloperLB so funny request...

**Comment by AndroidDeveloperLB** [comment]
@hackware1993 No, it's a valid request. It's a view, so it should be possible to see it, or at least a placeholder for it, instead of seeing errors covering the entire UI designer and making it un-usable.

**Comment by hackware1993** [comment]
@AndroidDeveloperLB Oh, it seems that most third-party ui libs don't suport this.

**Comment by AndroidDeveloperLB** [comment]
@hackware1993 So tell them to support it. It's a basic thing to request for a custom view. Also shown in many tutorials .
